pkg/jobs/job_scheduler.go

```diff
@@ -477,7 +488,14 @@ var schedulerMaxJobsPerIterationSetting = settings.RegisterIntSetting(
 	settings.TenantWritable,
 	"jobs.scheduler.max_jobs_per_iteration",
 	"how many schedules to start per iteration; setting to 0 turns off this limit",
-	10,
+	5,
+)
+
+var schedulerSchedulePlanningTimeout = settings.RegisterDurationSetting(
+	settings.TenantWritable,
+	"jobs.scheduler.schedule_planning.timeout",
```

Comment:
nit: this isn't schedule planning. it's schedule _execution_ which involves _job_ planning.

Reply:
Done.